### PR TITLE
doc: show example of updating multiple values

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: diesel/src/query_builder/functions.rs [hunk 1]
```diff
@@ -37,6 +35,12 @@ use super::insert_statement::Insert;
 /// # #[cfg(not(feature = "postgres"))]
 /// # fn main() {}
 /// ```
+///
+/// To update multiple columns, give `set` a tuple argument:
+///
+/// ```no_run
```
Comment (killercup) [inline diesel/src/query_builder/functions.rs]: Setting this to `no_run` will still compile it (which fails)

Comment (tshepang) [inline diesel/src/query_builder/functions.rs]: oh, yes

File: diesel/src/query_builder/functions.rs [hunk 2]
```diff
@@ -37,6 +35,33 @@ use super::insert_statement::Insert;
...
+///
+/// ```rust
+/// # #[macro_use] extern crate diesel;
+/// # include!("src/doctest_setup.rs");
+/// #
+/// # table! {
+/// #     users {
+/// #         id -> Integer,
+/// #         name -> VarChar,
+/// #         surname -> VarChar,
```
Comment (killercup) [inline diesel/src/query_builder/functions.rs]: Sorry, but this users table doesn't have a surname column (see `doctest_setup.rs`).

Comment (tshepang) [inline diesel/src/query_builder/functions.rs]: ok, I might have a look at some point

**Comment by Eijebong** [comment]
This has been merged as part of #915 
Thank you @tshepang for the original implementation :)

**Pull request closed without merging**